shadow-dom/focus/focus-tabindex-order-shadow-varying-tabindex-2.html

```diff
@@ -49,18 +49,18 @@
   // <div #host3></div>
   //   #ShadowRoot
   //     <button>Button</button>
-  assert_equals(document.activeElement, host1);
-  assert_equals(root1.activeElement, root1.querySelector("button"));
+  assert_equals(document.activeElement, host1, 't1');
+  assert_equals(root1.activeElement, root1.querySelector("input"), 't2');
 
   await navigateFocusForward();
-  assert_true(forwarderFocused);
-  assert_equals(document.activeElement, host2);
-  assert_equals(root2.activeElement, root2.querySelector("button"));
+  assert_true(forwarderFocused, 't3');
+  assert_equals(document.activeElement, host2, 't4');
+  assert_equals(root2.activeElement, root2.querySelector("input"), 't5');
 
   // In buggy Firefox build, the following focus navigation will
   // move the focus back to #host1's button.
   await navigateFocusForward();
-  assert_equals(document.activeElement, host3);
-  assert_equals(root3.activeElement, root3.querySelector("button"));
+  assert_equals(document.activeElement, host3, 't6');
+  assert_equals(root3.activeElement, root3.querySelector("input"), 't7');
```

Comment:
the t1/t2/t3/t4/t5/t6/t7 messages are a bit mysterious. Can you make them more explicit or remove them>

Reply:
Yeah, I forgot to revert them after debugging. Will revert.